### PR TITLE
Seperate engine docs where referenced in #244

### LINKED ISSUE #244: Separate Engine from Enterprise docs
- [ ] Remove mention of Anchore Engine quickstart from https://docs.anchore.com/current/docs/quickstart/

- [ ] Remove "What is Anchore Engine" and "Anchore Enterprise vs Anchore Engine" comparison from https://docs.anchore.com/current/docs/overview/concepts/ [This question can be left in the FAQ]

- [ ] Remove Open Source section from the FAQ at https://docs.anchore.com/current/docs/faq/ and replace Anchore Engine with Anchore Enterprise throughout. 

- [ ] Replace Anchore Engine with Anchore Enterprise at https://docs.anchore.com/current/docs/overview/concepts/images/

- [ ] Replace Anchore Engine with Anchore Enterprise at https://docs.anchore.com/current/docs/overview/concepts/policy/bundles/

- [ ] Replace Anchore Engine with Anchore Enterprise at https://docs.anchore.com/current/docs/overview/requirements/

- [ ] https://docs.anchore.com/current/docs/engine/engine_installation/configuration/ needs to be copied to the Enterprise tree.

- [ ] https://docs.anchore.com/current/docs/engine/engine_installation/upgrade/ needs to be copied to the Enterprise tree.

- [ ] Replace Anchore Engine with Anchore Enterprise at https://docs.anchore.com/current/docs/data_privacy/

- [ ] Replace Anchore Engine with Anchore Enterprise at https://docs.anchore.com/current/docs/troubleshooting/

- [ ] Remove https://docs.anchore.com/current/docs/engine/ from tree and replace with a 301 to Github URL.

### PR DESCRIPTION
**Pull request by zburstein**
Fixes #244 bullet points by removing the specified references to Anchore Engine and removing the engine subdirectory in the docs

Signed-off-by: Zane Burstein <zane.burstein@anchore.com>

### CODE REVIEW

**Comment by zburstein** [comment]
I compiled a list of images that are diagrams that reference engine. I posted them in the issue: https://github.com/anchore/enterprise-docs/issues/244#issuecomment-733279345

**Comment by zburstein** [comment]
@zhill I got most of the references to engine but there are still a few remaining ones that I was not 100% sure how to proceed on that I might need feedback on in order to proceed:

- overview/requirements: explains the structure of enterprise and I am as of now unaware if any references to engine should remain from this
- overview/: explains the structure of enterprise and I am as of now unaware if any reference to engine should remain from this
- overview/air_gapped: references engine in detailing some information about structure. Unaware if any references to engine should remain
- installation/helm: references to engine helm that I am not sure if need to remain
- installation/helm/aks: references to engine helm that I am not sure if need to remain
- installation/helm/eks: references to engine helm that I am not sure if need to remain
- installation/helm/gke:  references to engine helm that I am not sure if need to remain
- installation/ui/ui_configuration/: reference to env variable that might need to remain
- installation/rbac/: appears to explicitly reference engine. Might need reworking
- installation/feeds/: appears to explicitly reference engine in context of how to use enterprise feeds with engine installation. Not sure if this needs to remain
- installation/feeds/migrating_data_between_feeds: not sure if this is an explicit reference to engine that might need to be preserved
- installation/storage/object_store/migration/: references an engine installation for example on how to migrate to new data drivers. Not sure if there are other considerations for changing this to enterprise
- using/ui_usage/policies/mappings/: references engine release version earlier than I can map to enterprise with existing documentation
- /using/cli_usage/policies/policy_gate_dockerfile/: references engine release version earlier than I can map to enterprise with existing documentation
- /using/cli_usage/feeds/feed_synchronization/: references engine release version earlier than I can map to enterprise with existing documentation
- using/cli_usage/event/: references engine release version earlier than I can map to enterprise with existing documentation
- /using/api_usage/: explicit references to engine that pertain to transition between engine and enterprise

File: content/_index.html [hunk 1]
```diff
@@ -23,43 +20,27 @@
...
 
 {{% blocks/feature svg="icons/jenkins-logo.svg" title="Jenkins" %}}
-Integrate Anchore Engine seamlessly into your container based CI/CD pipeline with our [Jenkins Plugin](https://plugins.jenkins.io/anchore-container-scanner) to include compliance, results visualizations, and reports to your workflow
+Integrate Anchore seamlessly into your container based CI/CD pipeline with our [Jenkins Plugin](https://plugins.jenkins.io/anchore-container-scanner) to include compliance, results visualizations, and reports to your workflow
 {{% /blocks/feature %}}
 
-{{% blocks/feature svg="icons/circleci-logo.svg" title="CircleCI" %}}
-Import our Partner-certified [CircleCI Orb](https://circleci.com/orbs/registry/orb/anchore/anchore-engine) to add Anchore Engine scanning to your CircleCI builds and automate policy-based compliance in your container deployments
+{{% blocks/feature svg="icons/gitlab-logo.svg" title="Gitlab" %}}
+Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed et justo id mi dictum ullamcorper vitae blandit quam. Pellentesque rutrum pellentesque libero, a elementum nisl laoreet a.
```
Comment (zburstein) [inline content/_index.html:31]: @nwl Here is the lorum ipsum you said I should add as a placeholder for the gitlab description

**Comment by zburstein** [comment]
@nwl @zhill  A point that I would like to highlight for review is how some of the folder structures should be handled for data that was moved from engine docs to enterprise docs as these structures affect the navigator in the sidebar.

For the 7th point in https://github.com/anchore/enterprise-docs/issues/244#issue-739477289 to move engine configuration into the enterprise docs, I just moved it over as a sub folder called `configuring_engine` that I placed within `content/docs/installation/configuration`. I preserved the references to engine in those file, but removed `content_hints.md` and `malware.md` because those were already present in enterprise: 
```
├── configuration
│   ├── _index.md
│   ├── configuring_engine
│   │   ├── _index.md
│   │   ├── config.md
│   │   ├── custom_certs.md
│   │   ├── network_proxies.md
│   │   ├── tls_ssl_config.md
│   │   ├── user_credential_storage.md
│   │   └── using_env_vars.md
│   ├── content_hints.md
│   └── malware.md
```

Should I unpack `configuring_engine` so there is only a single folder for configuration details that contains all of those files. Did I also maybe misunderstand the request? Should I also remove any mention of engine from those files moved over to enterprise? Still new and not fully up to speed on all the configuration and runtime differences between enterprise and engine so I might have done this one wrong


Additionally for point 8 in https://github.com/anchore/enterprise-docs/issues/244#issue-739477289 to copy engine upgrade docs to enterprise, I made `content/docs/installation/upgrade_engine/_index.md` and `content/docs/installation/upgrade_enterprise/_index.md`:
```
├── upgrade_engine
│   └── _index.md
└── upgrade_enterprise
    └── _index.md
```

 Should I instead make a single folder at `content/docs/installation/upgrade` and make separate md files for engine and enterprise upgrade within that folder as opposed to having two separate folders or sub folders? So something like this:
```
└── upgrade
    ├── _index.md
    ├── upgrade_engine.md
    └── upgrade_enterprise.md
```

**Comment by zhill** [comment]
I'll review w/local build to see how to handle the items you've identified @zburstein.

**Comment by nwl** [comment]
Re: upgrade section - if possible it would be good to have this as a single file rather than two separate ones. 

Also, can you please remove the "Anchore Enterprise vs. Anchore Engine" section from the Overview?

Re: configuration sections - I think we need a complete IA review but think we can leave as is for now.

**Comment by zhill** [comment]
Re: upgrade, yes should be a single file and a single process to 'upgrade enterprise'.

**Comment by zhill** [comment]
Closing since superceded by #258

**Pull request closed without merging**